.gitlab/pipelines/stages/tests/smoke_tests.sh

```diff
@@ -31,4 +33,4 @@ wait_for_smoke_tests_pod || {
 
 # Follow the logs. If the tests fail, the logs command will also fail.
 pod_name="$(smoke_tests_pod_name)"
-bazel run @kubectl//:kubectl -- logs --follow "${pod_name}" --namespace "${KUBECF_NAMESPACE}" smoke-tests-smoke-tests
+"${KUBECTL}" logs --follow "${pod_name}" --namespace "${KUBECF_NAMESPACE}" smoke-tests-smoke-tests
```

Comment:
Hmm didn't we actually tell `kubectl` to follow the one container? How did this even work?
```suggestion
"${KUBECTL}" logs --follow "${pod_name}" --namespace "${KUBECF_NAMESPACE}" --container smoke-tests-smoke-tests
```